README.md

```diff
@@ -1,43 +1,33 @@
 
 This repository also contains the Zowe Node Client SDK. The SDK lets you leverage the underlying APIs to build applications that interface with the mainframe.
 
-## Contents  <!-- omit in toc -->
+<br/>
 
- - [Early Access Features](#early-access-features)
+## **Contents**  <!-- omit in toc -->
  - [Documentation](#documentation)
- - [Contribution guidelines](#contribution-guidelines)
- - [Build Zowe CLI from source](#build-zowe-cli-from-source)
- - [Install Zowe CLI from source](#install-zowe-cli-from-source)
- - [Uninstall Zowe CLI](#uninstall-zowe-cli)
- - [Configure Zowe CLI](#configure-zowe-cli)
+ - [Contribution Guidelines](#contribution-guidelines)
+ - [Building Zowe CLI From Source](#build-zowe-cli-from-source)
+ - [Installing Zowe CLI From Source](#install-zowe-cli-from-source)
+ - [Uninstalling Zowe CLI](#uninstall-zowe-cli)
+ - [Configuring Zowe CLI](#configure-zowe-cli)
  - [Zowe Node Client SDK](#zowe-node-client-sdk)
- - [Run system tests](#run-system-tests)
+ - [Running System Tests](#run-system-tests)
  - [FAQs](#frequently-asked-questions)
 
-## Early Access Features
-
-To try out early access features, install the "next" release of Zowe CLI (`npm install -g @zowe/cli@next`). This version may receive breaking changes and is intended to gather early feedback on what may become a future LTS release.
-
-For documentation about these features, see these files:
-
-- [Using Global Profile Configuration](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md)
-- [Daemon Mode Design Overview](https://github.com/zowe/zowe-cli/blob/next/zowex/design.md)
-
-## Documentation
+<br/>
 
+## **Documentation**
 For detailed information about how to install, configure, and use Zowe CLI, see [Zowe CLI Documentation](https://docs.zowe.org/stable/getting-started/cli-getting-started/). The documentation includes examples and tutorials for how to contribute to Zowe CLI and develop CLI plug-ins.
 
-The `docs` directory in this repository contains auto-generated typescript documentation under the `docs/typedoc` directory. To access the typescript documentation locally, navigate to the local `node_modules` directory that contains the installed package and access the `docs` directory after you install the Zowe CLI package.
+To view the Web Help for all Zowe CLI commands and contributed plug-ins, see the [Zowe CLI Web Help](https://docs.zowe.org/stable/web_help/index.html). To view all locally accessible commands, run `zowe --help-web`. For more use cases and tutorials visit [Medium.com/zowe](https://medium.com/zowe).
```

Comment:
I suggest that we still point readers to the docs directory in addition to the other sources of documentation that you added.

Something like:

    Engineering design documentation is contained in the 'docs' directory in this repository.